content/nextflow-on-aws-batch/40_nextflow202/35_batch-squared-config.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Batch Sqared Config"
+title: "Create Job-Definition"
```

Comment:
No need for a dash '-' between Job and Definition

Reply:
done